.github/workflows/main.yml

```diff
@@ -41,6 +41,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
+        pip install setuptools_scm
```

Comment:
Do we need to add this step to the deploy script, as well? (Could failure to do so result in failed prod deployment?)

Reply:
Retroactively tagging @fgregg

Reply:
it wasn't present before, so i would wait to do that.